catalyst/contrib/criterion/ce.py

```diff
@@ -14,3 +14,31 @@ def forward(self, input, target):
         loss = -torch.sum(input * target)
         loss = loss / input.size()[0] if self.size_average else loss
         return loss
+
+
+class SymmetricCrossEntropyLoss(nn.Module):
+    """
+    Symmetric Cross Entropy
+    paper : https://arxiv.org/abs/1908.06112
+    """
+    def __init__(self, alpha, beta, num_classes):
+        super(SymmetricCrossEntropyLoss, self).__init__()
+        self.alpha = alpha
+        self.beta = beta
+        self.num_classes = num_classes
+
+    def forward(self, input, target):
```

Comment:
and the same thing here...
it would be also nice, if you could write the dimensions of the tensors, like
`input - [batch_size; num_features]` or something like that